test-e2e/tests/payments/subscription_test.ts

```diff
@@ -10,7 +10,7 @@ const jwProps: ProviderProps = {
   yearlyOffer: constants.offers.yearlyOffer.inplayer,
   paymentFields: constants.paymentFields.inplayer,
   creditCard: constants.creditCard.inplayer,
-  creditCardNamePresent: true,
+  creditCardFolderFieldName: constants.paymentFields.inplayer.creditCardFieldName,
```

Comment:
What does "Folder" mean here? Can we use something like 'cardholderFieldName' and keep it consistent throughout all uses?

Reply:
Also, why not add this to PaymentFields?

Reply:
Good eyes, @dbudzins! 😅